main.js

```diff
@@ -143,14 +143,18 @@ Promise.resolve().then(async () => {
     inactive: 'no'
   })
 
+  log(INFO, `OK, continuing with isView: ${isView}`)
+
   const initialModulePackageName = nameParamCase.startsWith(REACT_NATIVE_PREFIX)
     ? nameParamCase
     : REACT_NATIVE_PREFIX.concat(nameParamCase)
 
   const { modulePackageName } = await prompt({
     type: 'text',
     name: 'modulePackageName',
-    message: 'What is the full module package name?',
+    message: `What is the full native ${
+      isView ? 'view' : 'module'
+    } package name?`,
```

Comment:
I think it would be ideal to get this on one line, for example:

```js
  message: `What is the full native ${NATIVE_PACKAGE_TYPE} package name?`
```